orocos.osdeps

```diff
@@ -93,7 +93,10 @@ graphviz:
     macos-brew: graphviz
 
 gccxml:
-    debian,ubuntu: gccxml
+    debian: gccxml
```

Comment:
Damn. The hack in autoproj v1 does not work in v2, because v2 validates manifest.xml.